src/Raven.Server/Documents/Revisions/RevisionsStorage.cs

```diff
@@ -2630,14 +2627,14 @@ internal static unsafe Document TableValueToRevision(JsonOperationContext contex
                     Flags = TableValueToFlags((int)RevisionsTable.Flags, ref tvr),
                     TransactionMarker = TableValueToShort((int)RevisionsTable.TransactionMarker, nameof(RevisionsTable.TransactionMarker), ref tvr),
                     ChangeVector = TableValueToChangeVector(context, (int)RevisionsTable.ChangeVector, ref tvr),
-                    Data = new BlittableJsonReaderObject(tvr.Read((int)RevisionsTable.Document, out var size), size, context)
+                    Data = new CompressedBlittableJsonReaderObject(tvr.Read((int)RevisionsTable.Document, out var size), size, context, tvr.Id)
```

Comment:
I think it's valid scenario since we might set `DocumentsCompressionConfiguration.CompressRevisions  = false`

Reply:
changed `CompressedBlittableJsonReaderObject` to `DocumentBlittableJsonReaderObject` so now it can be both compressed and not compressed, so this should be fine